### PR TITLE
Changes from AngularDocFX_EN

### PR DESCRIPTION
**Pull request by ESShared**

### CODE REVIEW

File: en/components/datachart_series_tooltips.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ The following code snippet demonstrates how to add an item tooltip layer to the
         [xAxis]="xAxis"
         [yAxis]="yAxis"
         valueMemberPath="Value"
-        showDefaultTooltip=true>
+        [showDefaultTooltip]="true">
```
Comment (HUSSAR-mtrela) [inline en/components/datachart_series_tooltips.md]: why are you using [] for showDefaultTooltip?

**Comment by HUSSAR-mtrela** [comment]
@zdrawku @pamelabrasil 
I don't get why do we want to remove the lazy loading functionality for Gauges and BulletGraph?

File: en/components/bulletgraph.md [hunk 2]
```diff
@@ -110,8 +110,8 @@ Performance value is the primary measure displayed by the component and it is vi
 </igx-bullet-graph>
 ```
 
-<div class="sample-container loading" style="height: 125px">
-    <iframe id="bullet-graph-measures-iframe" data-src='{environment:demosBaseUrl}/gauges/bullet-graph-measures' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+<div class="sample-container" style="height: 125px">
+    <iframe id="bullet-graph-measures-iframe" src='{environment:demosBaseUrl}/gauges/bullet-graph-measures' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
 </div>
```
Comment (HUSSAR-mtrela) [inline en/components/bulletgraph.md]: who is making these changes and why?

Comment (zdrawku) [reply en/components/bulletgraph.md]: As I see these changes are comming from `tfsbuild` [commit 120bbdac42e5f663ea725fffa1d11a12d25e8eb4](https://github.com/IgniteUI/igniteui-docfx/pull/1156/commits/120bbdac42e5f663ea725fffa1d11a12d25e8eb4).

As I've said, they are not wrong, its just they remove the lazy loading functionality of the iframe. If you want we can leave these changes, it is fine by me.

File: en/components/datachart_series_tooltips.md (hunk 1 above)
Comment (zdrawku) [reply en/components/datachart_series_tooltips.md]: Who are you referring to? There is no need to use brackets, if `showDefaultTooltip` input was using a property, then brackets should be used in order to bind `showDefaultTooltip` to it.

**Comment by zdrawku** [comment]
As we discussed with @pamelabrasil, we are not going to merge PRs related to Charts, Gauges and Spredsheets for the time being. Once we promote current version of staging to prod I will remove `do not merge` label.

**Comment by pamelabrasil** [comment]
removing the do not merge label as it was pushed to production today

**Pull request merged**